data/logs_model/document_logs_model.py

```diff
@@ -511,9 +511,15 @@ def log_action(
         try:
             self._logs_producer.send(log)
         except LogSendException as lse:
-            strict_logging_disabled = config.app_config.get("ALLOW_PULLS_WITHOUT_STRICT_LOGGING")
-            logger.exception("log_action failed", extra=({"exception": lse}).update(log.to_dict()))
-            if not (strict_logging_disabled and kind_name in ACTIONS_ALLOWED_WITHOUT_AUDIT_LOGGING):
+            strict_logging_disabled = config.app_config.get("ALLOW_WITHOUT_STRICT_LOGGING") or (
+                config.app_config.get("ALLOW_PULLS_WITHOUT_STRICT_LOGGING")
```

Comment:
do we need two config options? Can we just use `ALLOW_WITHOUT_STRICT_LOGGING` ?

Reply:
That's an existing config option that controls the behavior of the registry when it is in read-only mode. That's a different aspect than tolerating logging failures for everything to happen. I am undecided if we should remove that existing option, it might violate user expectations.